src/SFA.DAS.ApplyService.Web/Services/RoatpTaskListWorkflowService.cs

```diff
@@ -1,17 +1,15 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.Linq;
 using MoreLinq;
-using NPOI.POIFS.Storage;
 using SFA.DAS.ApplyService.Domain.Entities;
 using SFA.DAS.ApplyService.Web.Configuration;
 
 namespace SFA.DAS.ApplyService.Web.Services
 {
     public class RoatpTaskListWorkflowService: IRoatpTaskListWorkflowService
```

Comment:
The methods defined in this class look like they should be static

Reply:
Agreed, converted to static and removed DI